charts/minecraft/templates/extraports-svc.yaml

```diff
@@ -0,0 +1,42 @@
+{{- $minecraftFullname := include "minecraft.fullname" . }}
+{{- range .Values.minecraftServer.extraPorts }}
+{{- if default "" .service.enabled }}
```

Comment:
While testing the `.service.annotations` below I misconfigured my values structure with:
```yaml
minecraftServer:
  extraPorts:
    - enabled: true
      annotations:
        anno1: value1
        anno2: value2
```

and got
```
Error: template: minecraft/templates/extraports-svc.yaml:3:15: executing "minecraft/templates/extraports-svc.yaml" at <.service.enabled>: nil pointer evaluating interface {}.enabled
helm.go:76: [debug] template: minecraft/templates/extraports-svc.yaml:3:15: executing "minecraft/templates/extraports-svc.yaml" at <.service.enabled>: nil pointer evaluating interface {}.enabled
```

I now see why the helm template guide recommends flatter value structures. So, in this case rather than `.service.something` variables, it's better to do `.serviceSomething`